### ISSUE TITLE
Loading model with custom loss  in coreml

### ISSUE THREAD

**Issue by actionLUO** [akirasosa/mobile-semantic-segmentation#29]
I use
def matting_loss(x):

    def loss(y_true, y_pred):

       La = dice_coef_loss(y_true, y_pred)

       mul_true = K.concatenate([y_true, y_true, y_true], axis=-1) * x

       mul_pred = K.concatenate([y_pred, y_pred, y_pred], axis=-1) * x

       Lcolor = K.mean(K.sqrt(K.square(mul_true - mul_pred) + K.epsilon()))

       return La + 0.5 * Lcolor
   return loss
in keras, but cannot convet it to coreml, anyone who can help me!!!

**Comment by akirasosa** [comment]
I have rewrite code using PyTorch. Here is the current code to convert trained model into CoreML.

https://github.com/akirasosa/mobile-semantic-segmentation/blob/master/coreml_converter.py

Probably, you can convert without any problem, even if you use custom loss. If you have any problem, please let me know.

Thanks,
